src/sensors/odometer/interrupt/DirectionalOdometer.cpp

```diff
@@ -1,6 +1,7 @@
 #include <limits.h> // NOLINT(modernize-deprecated-headers)
 
 #include "DirectionalOdometer.hpp"
+#include <limits.h>
```

Comment:
:warning: **modernize-deprecated-headers** :warning:
inclusion of deprecated C++ header 'limits.h'; consider using 'climits' instead
```suggestion
#include <climits>```